### DISCUSSION TITLE
Does this tool support incremental update of the vul-db files ?

### DISCUSSION THREAD

**Discussion by WestFarmer** [aquasecurity/trivy#7546] [Q&A]
### Question

for  china users, network speed to ghcr.io is slow and unstable.

### Target

None

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment]
No, but you can host the database yourself.
https://aquasecurity.github.io/trivy/v0.55/docs/advanced/air-gap/
